survey-runner-database/global_vars.tf

```diff
@@ -21,6 +21,7 @@ variable "vpc_id" {
 }
 
 variable "private_route_table_ids" {
+  type = "list"
   description = "Route tables with route to NAT gateway"
```

Comment:
We need to change the scripts that pass lists to terraform to this format:
`-var 'private_route_table_ids=["<id_1>","<id_2>","<id_3>"]'`